code/src/App.js

```diff
@@ -1,9 +1,135 @@
-import React from 'react'
+import React, { useState } from 'react'
+
+const ageGroups = ["<20", "21-40", "40+"]
 
 export const App = () => {
+  const [personName, setPersonName] = useState("");
+  const [skiing, setSkiing] = useState(false);
+  const [surfing, setSurfing] = useState(false);
+  const [cycling, setCycling] = useState(false);
+  const [otherSport, setOtherSport] = useState(false);
+  const [ageGroup, setAgeGroup] = useState("");
+  const [gender, setGender] = useState("");
+
+
   return (
     <div>
-      Find me in src/app.js!
+      <div id="survey">
+
+      <form
+        className="question-form"
+         //onLoad={event => document.getElementById('review').style.visibility = 'hidden'}
+        onSubmit={(event) => {
+          event.preventDefault();
+          document.getElementById('survey').style.display = 'none';
+          document.getElementById('review').style.visibility = 'visible'}
+        }
```

Comment:
Try to stick to "React solutions" if possible.
For example add a state that takes care of submitted true or false and that renders components or not depending on that.